### ISSUE TITLE
10.14.3

### ISSUE THREAD

**Issue by colehellman** [tctien342/Asus-Vivobook-S510UA-Hackintosh#18]
Will this work with the Mojave Installer for 10.14.3?

**Comment by arstasczak** [comment]
Insalled latest Mojave on S510UN with 308 BIOS with EFI from this comment: https://github.com/tctien342/Asus-Vivobook-S510UA-High-Sierra-10.13-Hackintosh/issues/16#issuecomment-465977095

Everything works well :)

**Comment by colehellman** [comment]
My BIOS is 300.  Should I update or leave it be?

edit: also, My Windows partition does not show up on boot in clover. Any ideas would be much appreciated.

**Comment by arstasczak** [comment]
Update your bios. 

If you want to keep Windows try this tutorial: 
[https://www.tonymacx86.com/threads/add-windows-10-boot-efi-after-installed-os-x-clover-uefi-simple.182919/](url)

**Comment by colehellman** [comment]
I cannot get iMessage to work or sign into the App Store, but iCloud seems to work. iMessage says an error occurred during activation.

edit: I believe my iMessage and App Store logins aren't working bc I am using a USB adapter for wifi while I wait on my internal adapter to arrive from Newegg.

**Comment by solirjin** [comment]
your built-in microphone works? mine seem not working

**Comment by solirjin** [comment]
> I cannot get iMessage to work or sign into the App Store, but iCloud seems to work. iMessage says an error occurred during activation.
> 
> edit: I believe my iMessage and App Store logins aren't working bc I am using a USB adapter for wifi while I wait on my internal adapter to arrive from Newegg.

appstore works, imess and facetime don't , i also use usb wifi=))

**Comment by colehellman** [comment]
@solirjin What settings on your config.plist did you change to get the App Store to work?

**Comment by colehellman** [comment]
Updated my wireless card to the dw1560. Still cannot get some iCloud services to work. I am still unable to log in to the App Store. Any pointers would be much appreciated. 

edit: my exact model is F510UA-AH51

**Comment by solirjin** [comment]
> Updated my wireless card to the dw1560. Still cannot get some iCloud services to work. I am still unable to log in to the App Store. Any pointers would be much appreciated.
> 
> edit: my exact model is F510UA-AH51

https://hackintosher.com/forums/thread/generate-your-own-hackintosh-serial-number-board-serial-number-uuid-mlb-rom-in-clover.306/ follow this

**Comment by colehellman** [comment]
Still did not work, unfortunately. I also tried the idiot's guide to iMessage, and that didn't work either.

**Comment by solirjin** [comment]
> Still did not work, unfortunately. I also tried the idiot's guide to iMessage, and that didn't work either.

https://www.youtube.com/watch?v=IYismHReQ8A follow this, you may access to appstore and icloud, not imess and facetime

**Comment by colehellman** [comment]
I replaced my CLOVER folder with the newest one referenced in another issue thread. That got iMessage working. Then I ran this command from terminal: 

`defaults write com.apple.appstore.commerce Storefront -string "$(defaults read com.apple.appstore.commerce Storefront | sed s/,8/,13/)"`

After a reboot, I was able to log in to the App Store.  Now my Hackintosh laptop is almost perfect!

**Comment by solirjin** [comment]
> I replaced my CLOVER folder with the newest one referenced in another issue thread. That got iMessage working. Then I ran this command from terminal:
> 
> `defaults write com.apple.appstore.commerce Storefront -string "$(defaults read com.apple.appstore.commerce Storefront | sed s/,8/,13/)"`
> 
> After a reboot, I was able to log in to the App Store. Now my Hackintosh laptop is almost perfect!

can you send me that latest EFI folder ?

**Comment by colehellman** [comment]
@solirjin [Clover](https://github.com/tctien342/Asus-Vivobook-S510UA-High-Sierra-10.13-Hackintosh/files/2889166/CLOVER.zip) folder mentioned in the huge issue thread that helped get me up and running.  Never ended up updating by BIOS from 300 because currently I cannot boot into windows.  Still trying to figure that one out.  Make sure to replace the Serial Number, MLB, and UUID in the config.plist.

edit: for clarity, I replaced my wifi adapter with the dw1560 as suggested in the readme

**Comment by solirjin** [comment]
> @solirjin [Clover](https://github.com/tctien342/Asus-Vivobook-S510UA-High-Sierra-10.13-Hackintosh/files/2889166/CLOVER.zip) folder mentioned in the huge issue thread that helped get me up and running. Never ended up updating by BIOS from 300 because currently I cannot boot into windows. Still trying to figure that one out. Make sure to replace the Serial Number, MLB, and UUID in the config.plist.
> 
> edit: for clarity, I replaced my wifi adapter with the dw1560 as suggested in the readme

I saw, but how do you activate imess ? I just know use icloud only with enough serial number, UUID

**Comment by colehellman** [comment]
@solirjin I believe I am able to use iMessage because I have an internal wifi card.  You may need to trick your mackintosh into thinking your wifi adapter is internal.
